### ISSUE TITLE
Can't change into directory with space in name

### ISSUE THREAD

**Issue by thorstenkampe** [nickcox/cd-extras#9]
```
[~] pwsh> cd '.\directory with'

cmdlet Transpose-Location at command pipeline position 1
Supply values for the following parameters:
With:
```
It looks llike the problem is confined to directory names consisting of two words. The heuristic mentioned in #4 seems not to work properly.

**Comment by thorstenkampe** [comment]
Wouldn't it make sense to first try to change into the directory given as argument to `cd`  (or see if it exists) and then try transpose?

**Comment by thorstenkampe** [comment]
It seems the problem is with the dispatching in `PostCommandLookup.ps1`. Please see this diff:
``` diff
@@ -18,6 +18,3 @@
         # two arg: transpose
-        if (
-          @($arg).Length -eq 2 -and
-          @($params).Length -eq 0 -and
-          (-not ($args -match '^(/|\\)'))) {
+        if ($args.Length -eq 2) {
           Transpose-Location @args
```

**Comment by nickcox** [comment]
Should be fixed in [0.2.2](https://www.powershellgallery.com/packages/cd-extras/0.2.2).

I think I'm going to need to write some tests to protect against these types of regressions.
